appdatabase/database.go

```diff
@@ -132,6 +132,13 @@ func FixMissingKeyUIDForAccounts(sqlTx *sql.Tx) error {
 		log.Error("Migrating accounts: failed to update key_uid/derived_from", "err", err.Error())
 		return err
 	}
+	// fix the default wallet account color issue https://github.com/status-im/status-mobile/issues/20476
+	// we don't care the other type of account's color
+	_, err = sqlTx.Exec(`UPDATE accounts SET color = 'blue' WHERE wallet = 1`)
```

Comment:
@qfrank is this an issue only for v1 accounts or all accounts?
If the latter, this looks like a perfect candidate for a migration script. Why it's done here?